src/persist-client/src/operators/shard_source.rs

```diff
@@ -547,7 +580,7 @@ where
                     };
 
                     let retired_parts = inflight_parts.drain_filter_swapping(|(upper, _size)| {
-                        PartialOrder::less_equal(&*upper, &flow_control_upper)
+                        PartialOrder::less_equal(&refine_antichain(upper), &flow_control_upper)
```

Comment:
This looks like an interesting line to test, seems uncovered.

Reply:
This is unfortunately a codepath that is already untested in ci, as the default non-granular backpressure config (in prod and in ci) is like 4TB

Reply:
I know, my argument is that changing it without having a test increases risk. Can we reduce the backpressure config to test this explicitly?